required/latex-lab/latex-lab-math.dtx

```diff
@@ -287,197 +288,334 @@
+%    the use of \texttt{Alt} does not shadow mathml). It can be enabled for
+%    all engines and PDF versions.
+%   \item[enable/disable] \verb+\tagpdfsetup{math/alt/use}+ (local boolean,
+%   so can be used on individual equations)
+%   \item[default value] A template text (stored in \cs{l_@@_content_template_tl})
+%   starting with \texttt{LaTeX formula starts}.
+%   \item[user value] No interface currently provided. This needs optional arguments
+%   or an external setup command.
+%   See \url{https://github.com/latex3/tagging-project/discussions/717}.
+%
+%   \end{description}
+%
+%  \item[source-AF] The \LaTeX{}-source of the equation can
+%  be attached as an associated file with mime-type
+%  application/Fx-tex. The \texttt{AFRelationship} is \texttt{Source}.
+%  The source is embedded without expansion. This means that targets of
+%  references and macros are not resolved.
+%  The files are by default not shown in the EmbeddedFiles pane,
+%  this can be enabled with |viewer/pane/mathsource=true|.
+%  If a A-standard is used, it must be one that allows embedded files.
+%
+%  \begin{description}
+%   \item[enabled by default?] true for all engines and PDF versions
+%   \item[enable/disable] \verb+\tagpdfsetup{math/tex/AF}+ (local boolean, so can
+%   be used on individual equations)
+%   \item[default value] source code including dollars or environment name.
+%   \item[consumer support] Currently only ngpdf makes
+%  use of it: if there is no mathml it passes the source to mathjax.
+%   \end{description}
+%
+%  \item[luamml] The following options make (with lualatex) use
+%  of the \pkg{luamml} package. \pkg{luamml} is currently automatically
+%  loaded (at the end of the preamble) if \pkg{unicode-math} has been detected.
+%  The loading can be forced or suppressed
+%  with \verb+\tagpdfsetup{math/mathml/luamml/load=true/false}+.
+%
+%  \pkg{luamml} affects all math, locally it can be stopped with |math/mathml/ignore|,
+%  or by using the commands described in the package.
+%
+%  \item[mathml-AF] A mathml representation of the equation can be attached
+%  to the structure. The options are rather complex as the
+%  keys have to control three different  tasks:
+%  The \emph{generation} of the file with the mathml fragments,
+%  the \emph{reading} and \emph{embedding} of the mathml fragments,
+%  and the \emph{association} of a mathml fragment to a specific equation.
+%
+%  \begin{description}
+%  \item[generation]
+%   With pdf\LaTeX{} mathml fragments can not be generated automatically,
+%   but a file with dummy fragments for every equation will be written if
+%   \verb+\tagpdfsetup{math/mathml/write-dummy}+ is issued.
+%
+%   With lua\LaTeX{} a file with mathml fragments will be created automatically
+%   if the package \pkg{luamml} has been loaded (see above).
+%
+%   \item[reading and embedding]
+%    By default the code will read and embed
+%    mathml from |\jobname-mathml.html| and |\jobname-luamml-mathml.html|.
```

Comment:
what does it do when both are present and in which order?

Reply:
the first wins. I added a clarification.